### PR TITLE
docs: make accessibility tests part of the integration test

### PR DESCRIPTION
**Pull request by barmac**
image

Related to https://github.com/bpmn-io/internal-docs/issues/909

### CODE REVIEW

**Comment by barmac** [comment]
We could remove `react-axe` as a follow-up. These tools duplicate each other.

File: docs/.project/INTEGRATION_TEST.md [hunk 1]
```diff
@@ -120,6 +121,11 @@ Based on the [test diagram](./test.bpmn.png):
 
 - [ ] Open [`broken.bpmn`](./broken.bpmn) and verify a proper error message is shown (_No diagram to display_)
 
+#### Accessibility
+
+- [ ] No new issues were reported in the devtools console
+- [ ] Running `window.__A11Y_CHECKER__.getViolations()` returns an empty array
```
Comment (nikku) [inline docs/.project/INTEGRATION_TEST.md:127]: I think this could be a button in the UI?

Comment (barmac) [reply docs/.project/INTEGRATION_TEST.md:127]: I could add it, sure.

Comment (barmac) [reply docs/.project/INTEGRATION_TEST.md:127]: https://github.com/bpmn-io/camunda-modeler-plugin-a11y-checker/issues/1

**Pull request merged**